src/browser_action/popup.js

```diff
@@ -68,14 +68,14 @@ class Popup {
       local: this._metrics.cls.value,
       background: this.background
     });
-    this.metrics.fid = new FID({
-      local: this._metrics.fid.value,
-      background: this.background
-    });
     this.metrics.inp = new INP({
       local: this._metrics.inp.value,
       background: this.background
     });
+    this.metrics.fid = new FID({
+      local: this._metrics.fid.value,
+      background: this.background
+    });
```

Comment:
Should this be moved to the end as it's deprecated? Maybe with a TODO comment to remove in future?